src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,11 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(1.2);
+      transition: transform 0.3s color 0.3s;
```

Comment:
Move transition on whole class rather than on :hover. Than your animation will be smooth both ways.